doc/sphinx/compiling.rst

```diff
@@ -5,21 +5,22 @@ Cantera Compilation Guide
 *************************
 
 .. toctree::
-   :hidden:
 
    SCons Configuration Options <configuring>
 
 This guide contains instructions for compiling Cantera on the following
 operating systems:
 
-* Linux
+* `Linux`_
 
-  * Ubuntu 12.04 LTS (Precise Pangolin) or newer; 16.04 LTS (Xenial Xerus) or
-    newer is recommended
-  * Debian 7.0 (Wheezy) or newer; 8.0 (Jessie) or newer is recommended
-  * Fedora 20 or newer; 22 or newer is recommended
+  * :ref:`sec-ubuntu-debian-reqs`
+  * :ref:`sec-fedora-reqs`
+  * :ref:`sec-opensuse-reqs`
+
+* `Windows`_
+
+  * :ref:`sec-windows-reqs`
 
-* Windows 7 or newer (32-bit or 64-bit versions)
 * OS X 10.9 (Mavericks) or newer; 10.10 (Yosemite) or newer is recommended
```

Comment:
My only experience with this is via TeX, which had to make some changes to the installer for Mac when El Capitan came out: http://tex.stackexchange.com/a/249967/32374 To be honest, I'm not sure of all of the implications, which is one reason why I just want to avoid it.